docs/src/pages/layout/breakpoints/UseWidth.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import useWidth from '@material-ui/core/useWidth/unstable_useWidth';
```

Comment:
`import { unstable_useWidth as useWidth } from '@material-ui/core/useWidth';` to replicate the docs?